test/manager/bundler/extract.spec.ts

```diff
@@ -33,12 +66,85 @@ describe('lib/manager/bundler/extract', () => {
       platform.getFile.mockReturnValueOnce(railsGemfileLock);
       const res = await extractPackageFile(railsGemfile, 'Gemfile');
       expect(res).toMatchSnapshot();
+      // couple of dependency of ruby rails are not present in the lock file. Filter out those before processing
+      expect(
+        res.deps
+          .filter(dep => {
+            return Object.prototype.hasOwnProperty.call(dep, 'lockedVersion');
```

Comment:
These should be filtered by name. Otherwise the code could return no locked versions at all and still pass this test